packages/axios-extension/src/abap/app-index-service.ts

```diff
@@ -34,4 +35,29 @@ export abstract class AppIndexService extends Axios implements Service {
         const response = await this.get('/', { params });
         return JSON.parse(response.data).results as AppIndex;
     }
+
+    /**
+     * Gets the URL to the app manifest with the specified id.
+     *
+     * @param {string} appId - The id of the app.
+     * @returns {Promise<string>} - URL to the app manifest.
+     */
+    public async getManifestUrl(appId: string): Promise<string> {
```

Comment:
I understand that this is fully sufficient for your use case but the call returns so many more important information that I am sure we need in other places as well, therefore, could you please enhance that function to be something like `getAppInfo` and return an object of type
```typescript
interface AppInfo {
   name: string;
   url: string;
   manifestUrl: string // or manifest, whatever is the newer name of the property and you take care of the mapping here;
  asyncHints: Array<...>;
}
```

Reply:
Good point, the implementation is now enhanced as per your recommendation with: 3d7fc54d7441290ff13f8020f49c734e930b5c02